multipaz-dcapi/src/androidMain/kotlin/org/multipaz/digitalcredentials/DigitalCredentials.android.kt

```diff
@@ -133,7 +133,7 @@ private suspend fun updateCredman() {
         .addOnFailureListener { Logger.i(TAG, "CredMan registry failed $it") }
 }
 
-internal suspend fun calculateCredentialDatabase(
+suspend fun calculateCredentialDatabase(
```

Comment:
This should remain internal. Once you move the tests for the DC implementations to multipaz-dcapi this will be possible.